pkg/deviceclaimingserver/enddevices/ttjsv2/util_test.go

```diff
@@ -170,7 +170,7 @@ func (srv *mockTTJS) handleClaim(w http.ResponseWriter, r *http.Request) { //nol
 		}
 
 		if dev.claimAuthenticationCode != req.OwnerToken {
-			writeResponse(w, http.StatusUnauthorized, "Owner token mismatch")
+			writeResponse(w, http.StatusForbidden, "claim failed with given owner token")
```

Comment:
@johanstokking to trigger the error `claim failed with given owner token` means sending a wrong token or it should be able to authenticate with the token just the actual claiming should fail?

Reply:
See below